development/jest.config.js

```diff
@@ -1,11 +0,0 @@
-module.exports = {
-  displayName: '/development',
-  collectCoverageFrom: ['<rootDir>/build/transforms/**/*.js'],
-  coverageDirectory: '../coverage',
-  coverageReporters: ['json'],
-  resetMocks: true,
-  restoreMocks: true,
-  testEnvironment: 'node',
```

Comment:
It looks like you've changed these tests to run on jsdom instead of node by removing this. Is that correct, and was that intentional?

Reply:
@Gudahtt This was intentional, as the tests also run successfully on jsdom, where the rest of our test suite is running. Is there any reason to keep it on node? If there is, then we could put the jest node environment comment on top of the two affected test files.

Reply:
The code under test here is run as part of our build system, so the Node.js environment is a better test environment because it matches the environment we'd be running the code in.

We could certainly deal with this using an environment comment. The comment doesn't seem ideal because then it would not be applied to new test modules, even though we'd want any new tests to be run in a Node.js environment (because this directory is used for development/build scripts, which are always run in Node.js). But maybe that's not a huge concern, we don't typically test our development/build scripts.

Reply:
I added the environment comments for now. In the future, if the number of development tests were to exponentially increase, we could reconsider running it with a separate configuration. A higher number of tests would justify spinning up a new machine to run the tests.